src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,17 @@
 #include "BatteryStateOfChargeService.h"
+#include <stdio.h>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    double ampHourTotal;
+    double ampHourUsed;
```

Comment:
Just to further expand on why, if there were multiple instances of BatteryStateOfChargeServices within the program then all of the instances would share the same ampHourUsed and ampHourTotal variable therefore would not work.